### PR TITLE
Mining Guide Updates

### PR DESCRIPTION
**Pull request by fosskers**
Commits from Rebecca that update the mining guide.

### CODE REVIEW

File: miner/README.org [hunk 1]
```diff
@@ -1,9 +1,9 @@
-#+TITLE: Mining for Chainweb
+#+TITLE: Mining for Kadena
```
Comment (fosskers) [inline miner/README.org]: Perhaps "for the Kadena Public Blockchain"?

File: miner/README.org [hunk 2]
```diff
@@ -25,24 +25,24 @@
...
+A blockchain is a series of /blocks/. Blocks contain /transactions/ that
 represent coin transfers or other Smart Contract interactions. Mining is the
 process of mathematically "solving" a block. Unless solved, a block cannot be
 included in the chain. Without mining, a blockchain cannot progress and
 transactions will never finalize.
 
-Economically, mining is the principal way to obtain currency on a Chainweb
-network. Since Chainweb uses Proof-of-Work, computational effort is rewarded
+Economically, mining is the principal way to obtain currency on a Kadena
+network. Since Kadena uses Proof-of-Work, computational effort is rewarded
```
Comment (fosskers) [inline miner/README.org]: Is the Chainweb name going away? It's unclear in this line if "Kadena" refers to the network, the protocol, the company, etc.

File: miner/README.org [hunk 3]
```diff
@@ -25,24 +25,24 @@
...
 with coins associated with each block you solve. The more computational power
 you have, the more likely you are to be successful at mining.
 
-Unlike blockchains that have a single chain, Chainweb has many. Everyone who
+Unlike blockchains that have a single chain, the Kadena public blockchain has many. Everyone who
```
Comment (fosskers) [inline miner/README.org]: Seven syllables in "Kadena public blockchain" is a lot more than the two from "Chainweb".

File: miner/README.org [hunk 4]
```diff
@@ -25,24 +25,24 @@
...
 chooses to mine can spread their effort across multiple chains, advancing each
 chain equally, thereby reducing competition and wasted effort.
 
-* Chainweb Mining via Official Kadena Software
+* Kadena Mining via Official Kadena Software
```
Comment (fosskers) [inline miner/README.org]: We could just remove the name here altogether, and make it:

> Mining via Official Kadena Software

Where here, "Kadena" refers to the company.

File: miner/README.org [hunk 5]
```diff
@@ -60,7 +60,7 @@ Pair before beginning.
 ** Obtaining a Key Pair
 
 The easiest way to obtain a Key Pair is via the [[https://pact.kadena.io/][Pact Web]] application. Keys can
-be generated by the /Wallet/ widget in the bottom-right. Once generated, take
+be generated by the /Wallet/ widget in the bottom right. Once generated, take
```
Comment (fosskers) [inline miner/README.org]: My intuition says that the hyphen is correct, but the internet seems unable to agree. :thinking:

File: miner/README.org [hunk 6]
```diff
@@ -92,7 +92,7 @@ Doing so, we'll see the following configuration of your "mining identity":
     predicate: keys-all
 #+end_src
 
-With values specific to you replaced above (and the rest of Chainweb configured
+With values specific to you replaced above (and the rest of the Kadena public blockchain configured
```
Comment (fosskers) [inline miner/README.org]: In this case, the software is called `chainweb-node`, and that's what we mean here. The "Kadena public blockchain" is the entire network that exists "in the cloud" as a result of everyone running `chainweb-node`s.

File: miner/README.org [hunk 7]
```diff
@@ -207,12 +207,12 @@ it bad luck.
...
 *** I specify ~--chain=...~ but am getting work for other chains. Why?
 
-It is fundamental to the design of Chainweb that chains cannot progress much
-further than their neighbour chains. It may be that by asking for ~--chain=9~,
-the Node couldn't find work to do! In this case, it falls back to picking a
-random chain. This balances the needs of the Miner, who may want a specific
-Chain to progress efficiently, with the needs of the network, which requires all
-chains to grow evenly.
+It is fundamental to the design of the Kadena public blockchain that chains
+cannot progress much further than their neighbor chains. It may be that by
```
Comment (fosskers) [inline miner/README.org]: "Neighbour" is International English spelling. Do we have company consensus on which to use?

File: miner/README.org (hunk 5 above)
Comment (mightybyte) [reply miner/README.org]: Space looks better to my eye.

File: miner/README.org (hunk 6 above)
Comment (mightybyte) [reply miner/README.org]: Many of the changes from "Chainweb" to "the Kadena public blockchain" here make sense to me as a branding move.  But this one in particular seems weird.

File: miner/README.org (hunk 7 above)
Comment (mightybyte) [reply miner/README.org]: `neighbour` is definitely less common in the United States.

File: miner/README.org (hunk 6 above)
Comment (fosskers) [reply miner/README.org]: Then to get rid of the ambiguity completely we can say `chainweb-node` directly.

File: miner/README.org (hunk 7 above)
Comment (fosskers) [reply miner/README.org]: Right - do we want US spellings or international ones?

File: miner/README.org (hunk 1 above)
Comment (rebecca-io) [reply miner/README.org]: Yes! Much better.

File: miner/README.org (hunk 2 above)
Comment (rebecca-io) [reply miner/README.org]: You are correct. Moving forward, we will no longer be using Chainweb for user/client-facing documents. We can use chainweb, but only when speaking about chainwebs in general. 

For line 34, how would you position that? 'Kadena public blockchain network'?

File: miner/README.org (hunk 3 above)
Comment (rebecca-io) [reply miner/README.org]: This is okay, as we are wanting to make the transition from 'Chainweb' to 'Kadena public blockchain.'

File: miner/README.org (hunk 4 above)
Comment (rebecca-io) [reply miner/README.org]: Love it. Let's move forward with that change.

File: miner/README.org (hunk 5 above)
Comment (rebecca-io) [reply miner/README.org]: From my understanding, hyphens are needed when one or two more words (meant to modify) are placed before a noun. 

**Ex**: Keys can be generated by the Wallet widget in the bottom right vs. The keys are generated by the Wallet widget on the bottom-right corner.

File: miner/README.org [hunk 8]
```diff
@@ -25,24 +25,25 @@
...
 process of mathematically "solving" a block. Unless solved, a block cannot be
 included in the chain. Without mining, a blockchain cannot progress and
 transactions will never finalize.
 
-Economically, mining is the principal way to obtain currency on a Chainweb
-network. Since Chainweb uses Proof-of-Work, computational effort is rewarded
-with coins associated with each block you solve. The more computational power
-you have, the more likely you are to be successful at mining.
+Economically, mining is the principal way to obtain currency on the Kadena
+network. Since we uses Proof-of-Work, computational effort is rewarded with
```
Comment (mightybyte) [inline miner/README.org]: `we uses`

**Pull request merged**